app/src/main/java/net/squanchy/tweets/TweetsPageView.java

```diff
@@ -69,32 +73,40 @@ protected void onAttachedToWindow() {
         initList();
     }
 
+    @Override
+    protected void onDetachedFromWindow() {
+        super.onDetachedFromWindow();
+        subscription.dispose();
+    }
+
     private void initList() {
```

Comment:
@fourlastor that's weird. From what I can see `LifecycleViews` are stored in an `ArrayList` in the `HomeActivity` which forwards those callback to each views in a for loop. So I'd just need to add the view to the list